deb-package/build

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+#
+# Builds a Debian package for MXNET
+#
+# Copyright sociomantic labs GmbH 2017.
+# Distributed under the Apache License, Version 2.0.
```

Comment:
@nemanja-boric-sociomantic it's the upstream project license, so we might as well play nice.  @leandro-lucarella-sociomantic already discussed this and approved the choice.

The alternative would be to use Boost and include the Boost license directly in the file (I really don't fancy confusing things by putting the boost license in a separate file in the project root).